tests/Unit/Evolution/Systems/CurvedScalarWave/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@
 set(LIBRARY "Test_CurvedScalarWave")
 
 set(LIBRARY_SOURCES
+  Test_Characteristics.cpp
```

Comment:
Alphabetize this (move it down one line)

Reply:
done